source/frontend/src/components/common/Section/Section.tsx

```diff
@@ -30,7 +30,7 @@ export const Section: React.FC<
   className,
   ...rest
 }) => {
-  const [isCollapsed, setIsCollapsed] = useState<boolean>(!initiallyExpanded && true);
```

Comment:
this was a strange one.

Reply:
yeah, changed it after some tests to make sure it does what we want